base/util.jl

```diff
@@ -393,7 +393,7 @@ See also `Base.winprompt` (for Windows) and `Base.getpass` for secure entry of p
 
 # Examples
 
-```julia-repl
+```jldoctest; setup = :(import Base)
```

Comment:
Same here, how can this be a doctest when it needs input. Also, why would `import Base` be needed?